app/models/task.py

```diff
@@ -3,4 +3,23 @@
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.id'), nullable = True)
+
+
+
+    # def __init__(self, title, description, completed_at):
+    #     self.title = title
+    #     self.description= description
+    #     self.completed_at = completed_at
+    # COLUMNS = ["title", "description", "completed_at"]
```

Comment:
If you'd like to add additional constructor-like methods to a model type, classmethods can be a good way to go. Because the Model type sort of takes over how our constructor runs, it can be tricky (though not impossible) to customize it. We would need to be sure to check the SqlAlchemy documentation to make sure we did everything it was expecting us to do.